packages/template-block/README.md

```diff
@@ -0,0 +1 @@
+# Frontify Block Example
```

Comment:
Please align with the other readme (title + description)

Reply:
Done